api/Element.json

```diff
@@ -6883,11 +6883,107 @@
                 "version_added": "45"
               },
               "safari": {
+                "version_added": "14"
```

Comment:
Tested 13.1 and 14.1 with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9636 and it's added in 14.1, so this is probably right.